.github/workflows/ci.yml

```diff
@@ -244,19 +244,32 @@ jobs:
           # the required artifacts instead.
           path: artifacts
 
-      - name: Generate final report
+      - name: Generate final simulation report
         shell: git-nix-shell {0} --pure
         run: |
           set +f # Enable globbing
           pdfunite artifacts/*/report.pdf report.pdf
 
-      - name: Upload report
+      - name: Upload simulation report
         uses: actions/upload-artifact@v3
         with:
           name: Clock Control Simulation Report
           path: report.pdf
           retention-days: 14
 
+      - name: Generate HITL based plots
+        shell: git-nix-shell {0} --pure
+        run: |
+          mkdir hitl-plots
+          cabal run elastic-buffer-sim:plot -- artifacts/Bittide.Instances.Tests.FullMeshHwCc.fullMeshHwCcTest/ila-data/probe_test_start/ hitl-plots Bittide_Instances_Tests_FullMeshHwCc_fullMeshHwCcTest_callistoClockControlWithIla
```

Comment:
Once @hiddemoll is done, this can be part of a post-processing step in Shake.

Reply:
I put here, because this step downloads all artifacts anyway.